components/data/data-sources/src/main/java/com/zaxxer/hikari/pool/InUseConnectionEntry.java

```diff
@@ -0,0 +1,43 @@
+package com.zaxxer.hikari.pool;
```

Comment:
Is there a special reason to use foreign package name? It will be a problem on publishing to Maven Central

Reply:
yes, package visibility of used  hikari classes

Reply:
more specifically - com.zaxxer.hikari.pool.ProxyConnection#getPoolEntry

Reply:
ok, thanks